.travis.yml

```diff
@@ -0,0 +1,44 @@
+matrix:
+  include:
+# first the python tests
+    - language: python
+      python: 2.7
+      install:
+        - pip install -r tests/requirements.txt
+      before_script:
+        - cd tests
+      script:
+        - pytest
+# next the python formatting
+    - language: python
+      python: 2.7
+      install:
+        - pip install yapf==0.24.0
```

Comment:
I picked 0.24.0 because that was what we observed CircleCI using when we had a mismatch before. Are you okay with just standardizing on 0.24.0 for now or do you want to bump it to 0.25.0?

I'm assuming that once we've picked a version for CI purposes, we'll probably need to pin our dev setups to that version as well.

Reply:
I agree that we should pin all systems to some version as you suggest. One of the nice things about using `venv`s is that we could, potentially have different versions for different branches.

That leaves us the option of revisiting the version issue at a later time.
So, in the future, for some significant branch change, we can upgrade to avoid becoming too "stale" on that branch.

Since I assume that we will be dropping Circle-CI, for this branch, let's try the latest version available on the BBB today. (0.25.0) One trial run should tell us if that is not a good idea.

Reply:
Just tested with yapf = 0.25.0.  Let's go with that one for this branch.

Reply:
Done.